content/20.treatments.md

```diff
@@ -348,6 +348,20 @@ A quicker treatment regimen with HCQ needs to be investigated to elucidate how t
 Overall, the study provides promising data, although all of the findings still need to be validated in independent population cohorts.
 Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.
 Further investigation is necessary to confirm whether the drug indeed ameliorates symptoms and reduces inflammatory response.
+A retrospective analysis [@doi:10.1101/2020.04.16.20065920] was conducted on the data from 368 SARS-CoV-2 patients across all United States Veteran Health Administration medical centers.
+The study retrospectively investigated the effects of HCQ (n=97), HCQ+AZ (Azithromycin) (n=113) and no HCQ (n=158) on 368 patients for the primary outcomes of death and the need for mechanical ventilation. 
+Standard supportive care was rendered to all patients.
+The study included only men, with a median age of 65 years.
+Women were excluded due to their considerably lower representation in the available data (N=17). 
+The rate of death in the HCQ-only treatment condition was 27.8%, compared to 22.1% in the HCQ+AZ and 14.1% in the no-HCQ cohort.
+The risk of death was higher in the HCQ-only group compared to the no-HCQ group (adjusted hazard ratio: 2.61, p<0.03).
+Further, the risk of ventilation was similar across all three groups: (adjusted hazard ratio: 1.43 (HCQ) and 0.43 (HCQ+AZ) compared to no HCQ).
+The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients.
+Also, the HCQ and HCQ+AZ conditions were not associated with any advantage in rates of mechanical ventilation. 
+The study had a few limitations.
+The study was not randomized and did not include placebo.
+Also, baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups, and all of these factors could potentially influence treatment outcome. 
+Further the authors acknowledge that the effect of the drugs might be different in females as well as in pediatric subjects as these subjects were not the part of the study.
```

Comment:
```suggestion
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
```